.github/release-drafter.yml

```diff
@@ -1,32 +1,32 @@
-name-template: 'v$RESOLVED_VERSION'
-tag-template: 'v$RESOLVED_VERSION'
+name-template: "v$RESOLVED_VERSION"
```

Comment:
I am not sure this is necessary plus now it's not consistent with the rest of our services

Reply:
@pperzyna ⬆️ ?

Reply:
It is just a visual studio YAML formatter, no difference - https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.quoted_strings

 It is the same as here: https://github.com/dataswift/HAT2.0/blob/main/.github/release-drafter.yml